app.py

```diff
@@ -1,15 +1,43 @@
-""" Start of the application"""
+"""Main application that subscribes to events."""
 
-from flask import Flask
+import os
+import ssl as ssl_lib
+import certifi
+import slack
+from emoji_message import EmojiMessage
 
-app = Flask(__name__)  # pylint: disable=invalid-name
 
+@slack.RTMClient.run_on(event='emoji_changed')
+def onboarding_message(**payload):
```

Comment:
Naming confused me for a second :sweat_smile: onboarding made me think of the old feature for when someone joined the workspace.

Reply:
DEFINITELY NOT COPY AND PASTED FROM TUTORIAL